src/components/dashboardContent/homeProfileContent/HouseProfileForm.js

```diff
@@ -1,15 +1,52 @@
-import React from 'react';
+import React, {useEffect} from 'react';
+import { connect } from 'react-redux';
+import { isEditing, saveHouseInfo, cancelSaveHouseInfo } from '../../../actions/houseActions';
+import { useInput } from '../../../helper-functions/form-logic-functions';
 
 function HouseProfileForm(props) {
+  const [content, setContent, setContentDirect] = useInput(props.content);
+  useEffect(()=> {
+    setContentDirect(props.content)
+  }, [props.content])
+  console.log(props.content, content, 23123)
+  const handleEdit = () => props.isEditing(props.field);
+  const handleCancel = () => {
+    props.cancelSaveHouseInfo(props.field);
+    // setContent({ target: { value: props.content } });
```

Comment:
Would maybe just clean up some of the commented code and the `console.log` calls